main.c

```diff
@@ -134,7 +138,8 @@ main(int argc, char *argv[])
 
 	/* Prompt for a password. */
 	if (readpass(&passwd, "Please enter passphrase",
-	    dec ? NULL : "Please confirm passphrase", 1))
+	    dec ? NULL : !devtty ? NULL :
```

Comment:
I'm not sure what the point of this is... if we're not using a tty, we won't be printing a prompt anyway, I think?